### DISCUSSION TITLE
Peter Lowe’s Ad and tracking server list - Not updating anymore

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#1947] [General]
Hello,

I have noticed for the last few hours that Peter Lowe’s Ad and tracking server list in the Filter Lists pane for uBO isn't updating anymore.

I get an icon on that line that says "A network error prevented the resource from being updated." When I click on the house icon on that line, it tried to load pgl.yoyo.org/adservers and the webpage doesn't work. I also noticed that when I click the eye icon, it brings up the actual filter list and it has not been updated in more than a month. On line 5 on the list, it says "last updated:   Tue, 21 Dec 2021 18:15:38 GMT".

Is this list no longer maintained or updated? At least it seems so to me due to website not being there anymore and the last update was more than a month ago. Does this list need to be removed from uBO?

Thank you.

**Comment by ghost** [comment]
@pgl ?

**Reply by ghost** [reply]
Here:

>\# last updated:   Thu, 20 Jan 2022 13:28:02 GMT

**Reply by ghost** [reply]
The one from 1.40.8 package is from Tue, 21 Dec 2021 18:15:38 GMT

**Reply by ghost** [reply]
Thank you for the quick reply. I am using the current dev build 1.40.9b3 and have tried to force refresh the entire filter list page a few times now. I still have the Dec 21 date in my list.

Sorry for not including the version and a bit more information in the initial report.

**Comment by gorhill** [comment]
Mine was updated on Jan 6, so probably just a temporary server issue -- [this was posted](https://twitter.com/pgl/status/1484515885712875524) two days ago.

**Comment by Procyon-b** [comment]
My version is `# last updated:   Thu, 06 Jan 2022 10:36:23 GMT` and was (forced) updated from the server `4d.21h.49m` ago

**Edit:** [google cache](https://webcache.googleusercontent.com/search?q=cache:https%3A//pgl.yoyo.org/adservers/) tells us that the server was accessible on `24 Jan 2022 02:21:53 GMT`

**Comment by pgl** [comment]
Hey there. The server is indeed down right now. Temporary issues, should be back soon.

**Comment by uBlock-user** [comment]
List has been updated now, seems fixed.

**Comment by pgl** [comment]
Yep, thanks @uBlock-user. It's up again.

**Comment by ghost** [comment]
Thank you to all for the help! :+1:
